elastic/kibana
# [ResponseOps][Flapping] Update rule specific flapping tooltip UI

## Summary
Updates the rule specific flapping UI according to the finalized figma designs: https://www.figma.com/design/eTr6WsKzhSLcQ24AlgrY8R/Flapping-per-Rule--%3E-%23294?node-id=5265-28064&node-type=instance&t=GX5pGpXe1blP0x0G-0

<img width="798" alt="Screenshot 2024-09-25 at 8 56 47 PM" src="https://github.com/user-attachments/assets/2655a3cd-8097-442e-a3c0-71ab40675e31">
<img width="588" alt="Screenshot 2024-09-25 at 9 23 30 PM" src="https://github.com/user-attachments/assets/e40b43be-f5a5-4d80-a35c-c3032e017b56">

### To test:
1. Turn `IS_RULE_SPECIFIC_FLAPPING_ENABLED` to `true`.
2. Navigate to rule create/edit flyout and you should see the new tooltips based on the figma designs linked above.

### Checklist
